docs/reference/modules/network.asciidoc

```diff
@@ -31,7 +31,9 @@ Defaults to `_local_`.
 In order to join a cluster, a node needs to know the hostname or IP address of
 at least some of the other nodes in the cluster.  This setting provides the
 initial list of other nodes that this node will try to contact. Accepts IP
-addresses or hostnames.
+addresses or hostnames.  If a hostname lookup resolves to multiple IP addresses --
```

Comment:
while `If a hostname lookup resolves to multiple IP addresses ... each IP will be used..` is true, how does it relate to round robin DNS ? it seems that one just returns one, albeit a different one on each resolving round?